app/assets/v2/css/ethhack2019.css

```diff
@@ -4,98 +4,41 @@ a {
 }
 
 .header{
-  background-color: #0d023b;
-  background-size: cover;
-  background-position: 0px 0px;
-  background-repeat: no-repeat;
-  position: relative;
+  background-position: 0px 50%;
+  background-attachment: fixed;
 }
 
-@media screen and (min-width: 550px) {
-  .pdesc {
-    font-size: 1rem !important
-  }
-  .pdesc2 {
-    font-size: 1rem !important
-  }
-  .pdesc3 {
-    font-size: 1rem !important
-  }
-  .topnav {
-    font-size: 1em !important
-  }
-  .part {
-    font-size: 3rem !important
-  }
+body {
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-@media screen and (min-width: 920px) {
-  .topnav {
-    font-size: 1em !important;
-  }
-  .ethsvg {
-    margin-top: 6rem !important
-  }
-  .ethtitle {
-    font-size: 4rem !important
-  }
-  .poweredby {
-    font-size: 1.5rem !important;
-    margin-top: 4rem !important;
-  }
-  .ethsvg2 {
-    margin-top: 0.7rem !important;
-  }
-  .btns{
-    width: 200px !important;
-    font-size: 1rem !important;
-  }
-  .btns-wrapper{
-    margin: 5rem !important
-  }
-  .part {
-    padding-top: 10rem !important;
-    font-size: 3.5rem !important;
-  }
-  .pbtn {
-    margin-top: 5rem !important;
-    margin-bottom: 7rem !important;
-  }
-  .pdesc {
-    font-size: 1.75rem !important;
-    padding-right: 10rem !important;
-    padding-left: 10rem !important;
-    padding-top: 4rem !important;
-  }
-  .pdesc2 {
-    font-size: 1.75rem !important;
-    padding-right: 10rem !important;
-    padding-top: 4rem !important;
-  }
-  .pdesc3 {
-    font-size: 1.75rem !important;
-    padding-right: 10rem !important;
-    padding-top: 4rem !important;
-    padding-bottom: 8rem !important;
-  }
-  .bottomnav {
-    padding: 5rem !important;
-  }
+h1,
+h2,
+h3,
+h4 {
+  font-family: Arial-Black, Helvetica, sans-serif;
```

Comment:
new font-family ? I thought we were using Muli throughout the site :P

Reply:
not in this design, is something separated, also without the top navbar and footer